scripts/profile_helper_gui

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python
 import cProfile
 
-from regolith.helper_gui_main import main
```

Comment:
please check this.  If you remove this, can you still run `profile_helper_gui` successfully at the command line?